groovy/gradle/src/org/netbeans/modules/gradle/spi/GradleFiles.java

```diff
@@ -90,10 +91,21 @@ public GradleFiles(File dir, boolean knownProject) {
         }
         projectDir = dir;
         rootDir = projectDir;
+        searchPropertyFiles();
```

Comment:
Probably it would be better to calculate the property files whenever the getPropertFiles() call is made, also it shall it shall not be stored as a member variable. It also shall return an empty array if the GradleFiles does not represent a project folder. Also the returned list shall be immutable.
It seems I've totally forgotten to implement that method. Thanks for doing that.